ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/dualstack/provision/cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - ref: ovn-conf 
+  - ref: ovn-conf-baremetal-dualstack
```

Comment:
if `ovn-conf-baremetal-dualstack` can be applied to other platforms, not only for baremetal, maybe we can rename the step to remove `baremetal` ?

Reply:
We already have a `ci-operator/step-registry/ovn/conf/dualstack/ovn-conf-dualstack-commands.sh` for some reason.
Different CIDRS maybe?  Have to ask the devs.

Reply:
I think `  - cidr: fd65:a1a8:60ad::/112` is vSphere IBM7 DHCPv6 specific.

Reply:
Yes this matches  https://github.com/rvanderp3/vcenter-cluster-deployment/blob/main/terraform/vDS/segment-bastion/dhcp.conf.tmpl#L5

```
dhcp-range=fd65:a1a8:60ad:271c::2,fd65:a1a8:60ad:271c::128,ra-names,slaac,64,3600
```

So I don't know why this was called baremetal, have to check git history.

Reply:
renamed

Reply:
@mkowalski Okay to rename this to `ovn-conf-vsphere-dualstack`?

Reply:
Too late to the party (PTO) but yeah, this config is specific for vSphere IBM7 environment because the machine network CIDR has to match with what the environment provides. Sorry for causing the confusion in the first place